tensor_comprehensions/pybinds/pybind_options.cc

```diff
@@ -123,7 +111,25 @@ PYBIND11_MODULE(mapping_options, m) {
              std::vector<uint64_t>& blockSizes) {
             instance.mapToBlocks(blockSizes);
           },
-          "The configuration of CUDA grid, i.e. the number of CUDA blocks along three dimensions. Must be\nwithin the range allowed by CUDA (maximum 2^31-1 for the first value and 65535 for the second and third)");
+          "The configuration of CUDA grid, i.e. the number of CUDA blocks along three dimensions. Must be\nwithin the range allowed by CUDA (maximum 2^31-1 for the first value and 65535 for the second and third)")
+      .def(
+          "matchLibraryCalls",
+          [](tc::CudaMappingOptions& instance, bool match) {
+            instance.matchLibraryCalls(match);
+          },
+          "Replace computation patterns with calls to highly optimized libraries (such as CUB, CUTLASS) when possible")
+      .def(
+          "fixParametersBeforeScheduling",
+          [](tc::CudaMappingOptions& instance, bool fix) {
+            instance.fixParametersBeforeScheduling(fix);
+          },
+          "Perform automatic loop scheduling taking into account specific tensor sizes.\nMay produce faster kernels but significantly increases compilation time.\n Note that the mapping will be performed for specific tensor sizes anyway")
+      .def(
+          "unroll",
+          [](tc::CudaMappingOptions& instance, uint64_t factor) {
+            instance.unroll(factor);
+          },
+          "Perform loop unrolling on the generated code and produce at most the given number of statements");
```

Comment:
Is this file automatically generated?

Reply:
no it's manual and part of the python bindings that @prigoyal did

Reply:
Then why did these entries get reordered by this commit?
That makes it more difficult to see what actually changed.